.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Documentation
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - "@typespec/**"
+  pull_request:
+    branches: ["main"]
+    paths:
+      - "@typespec/**"
```

Comment:
Changes should only be picked up from the main branch, not from pull requests. This is because we don’t want to update the documentation from pull requests, only when changes have been merged into main

Reply:
Yes! I added it just to test that this works.